config/locales/en.yml

```diff
@@ -253,6 +253,8 @@ en:
       checkbox_label: select order %{order_number} to be reconciled
       bulk_note_checkbox: Use Bulk Note
       bulk_note: Bulk Reconciliation Note
+      bulk_deposit_number: Bulk CRT Number
+      bulk_deposit_number_hint: Please enter the CRT Number from your GL008
```

Comment:
This makes wonder about using a variable like `!deposit_number_prefix!` which can be easily changed for UMass